### ISSUE TITLE
PiecewiseHermiteEvaluation and PiecewiseLinearEvaluation

### ISSUE THREAD

**Issue by adutfoy** [openturns/openturns#1081]
Hi
The documentation of the classes PiecewiseHermiteEvaluation and PiecewiseLinearEvaluation are uncomplete: there is no example, the arguments are not described. There is non illustration.

Anne

**Comment by jschueller** [comment]
not exactly: http://openturns.github.io/openturns/master/user_manual/_generated/openturns.PiecewiseHermiteEvaluation.html?highlight=piecewisehermiteevaluation

**Comment by sofianehaddad** [comment]
There are indeed small examples. IMO we should add a small description (formula) of the underlying methods
